Cloud Pak for Data/WML/notebooks/unstructured_image/Watson OpenScale Explanation for Image Multiclass.ipynb

```diff
@@ -0,0 +1,1006 @@
+       "\n",
+       "Note: Keras and TensorFlow versions supported by WML are: Keras 2.1.6 with TensorFlow 1.13 backend and Keras 2.2.4 with TensorFlow 1.14 backend. The latter combination is used in this notebook."
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "### 2.1 Creating a model"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "# note: Install these package if you are not running this notebook in IBM Watson Studio\n",
+       "#!pip install keras==2.2.5\n",
+       "#!pip install tensorflow==2.5.0"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {},
+      "outputs": [
+       {
+        "name": "stdout",
+        "output_type": "stream",
+        "text": [
+         "Collecting keras_sequential_ascii\n",
+         "  Using cached keras_sequential_ascii-0.1.1.tar.gz (3.5 kB)\n",
+         "  Preparing metadata (setup.py) ... \u001b[?25ldone\n",
+         "\u001b[?25hRequirement already satisfied: keras in /Users/neelima/.pyenv/versions/3.10.0/lib/python3.10/site-packages (from keras_sequential_ascii) (2.10.0)\n",
+         "Building wheels for collected packages: keras_sequential_ascii\n",
+         "  Building wheel for keras_sequential_ascii (setup.py) ... \u001b[?25ldone\n",
+         "\u001b[?25h  Created wheel for keras_sequential_ascii: filename=keras_sequential_ascii-0.1.1-py3-none-any.whl size=3063 sha256=70cc27936e045584de5c690e29dc13f22fb3d534a5c10643b2dbe2a37b8cc20f\n",
+         "  Stored in directory: /Users/neelima/Library/Caches/pip/wheels/5d/cd/68/ee9141223b2b7335e4214fd0d50ec5ee12cf0875e3c9062e2f\n",
+         "Successfully built keras_sequential_ascii\n",
+         "Installing collected packages: keras_sequential_ascii\n",
+         "Successfully installed keras_sequential_ascii-0.1.1\n",
+         "\n",
+         "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip available: \u001b[0m\u001b[31;49m22.3.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m23.1.1\u001b[0m\n",
+         "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
+        ]
+       }
+      ],
+      "source": [
+       "!pip install keras_sequential_ascii"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": 7,
+      "metadata": {},
+      "outputs": [
+       {
+        "ename": "ModuleNotFoundError",
+        "evalue": "No module named 'keras_sequential_ascii'",
+        "output_type": "error",
+        "traceback": [
+         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+         "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
```

Comment:
can you clear this cell output. Please check other cells output and clear if there are any errors